cli/cli_app.go

```diff
@@ -585,22 +608,24 @@ func setTerragruntInputsAsEnvVars(terragruntOptions *options.TerragruntOptions,
 	return nil
 }
 
-func runTerraformWithRetry(terragruntOptions *options.TerragruntOptions) error {
+func runTerraformWithRetry(terragruntOptions *options.TerragruntOptions) (error, string) {
 	// Retry the command configurable time with sleep in between
 	for i := 0; i < terragruntOptions.MaxRetryAttempts; i++ {
 		if out, tferr := shell.RunTerraformCommandWithOutput(terragruntOptions, terragruntOptions.TerraformCliArgs...); tferr != nil {
-			if out != nil && isRetryable(out.Stderr, tferr, terragruntOptions) {
-				terragruntOptions.Logger.Printf("Encountered an error eligible for retrying. Sleeping %v before retrying.\n", terragruntOptions.Sleep)
-				time.Sleep(terragruntOptions.Sleep)
-			} else {
-				return tferr
+			if out != nil {
```

Comment:
Is this change necessary? Or is it a left over due to earlier attempts to do the `init` retries here?

Reply:
Yes. I had an integration test failing without this. The standard error wasn't set if there was no error resulting in nil dereference.